### PR TITLE
Add up/right/left/down arrow keymaps, similar to kakoune

### PR DESCRIPTION
**Pull request by itzmjauz**

### CODE REVIEW

File: helix-term/src/keymap.rs [hunk 1]
```diff
@@ -278,6 +296,23 @@ pub fn default() -> Keymaps {
             key!('k') => commands::extend_line_up,
             key!('l') => commands::extend_char_right,
 
+            KeyEvent {
+                code: KeyCode::Left,
+                modifiers: KeyModifiers::NONE
+            } => commands::move_char_left as Command,
```
Comment (archseer) [inline helix-term/src/keymap.rs:302]: You should use the `extend_` methods here

File: helix-term/src/keymap.rs [hunk 2]
```diff
@@ -35,10 +35,10 @@ use std::collections::HashMap;
 //          f = find_char()
 //          g = goto (gg, G, gc, gd, etc)
 //
-//          h = move_char_left(n)
-//          j = move_line_down(n)
-//          k = move_line_up(n)
-//          l = move_char_right(n)
+//          h = move_char_left(n)   || arrow-left  = move_char_left(n)
```
Comment (archseer) [inline helix-term/src/keymap.rs:38]: You don't have to update this comment, but you can add these to https://github.com/helix-editor/helix/blob/master/book/src/keymap.md :)

**Review by archseer — CHANGES_REQUESTED** [review]

File: helix-term/src/keymap.rs (hunk 1 above)
Comment (itzmjauz) [reply helix-term/src/keymap.rs]: My bad

File: helix-term/src/keymap.rs (hunk 2 above)
Comment (itzmjauz) [reply helix-term/src/keymap.rs:38]: I imagine in the future view-mode would probably also have arrow keys with similar functionality to j and k included.

**Pull request merged**